form.js

```diff
@@ -119,6 +120,17 @@ define('form', ['doc'], function($) {
 			}
 		});
 
+		form.find('[type="url"]').each(function(element){
+			var pattern = new RegExp(/^http(s)?:\/\/(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/),
```

Comment:
What about using a more complete regex, like [this one](http://stackoverflow.com/a/190405) (citing the source, of course)?

@luiz